ADOPTERS.MD

```diff
@@ -0,0 +1,15 @@
+Beckhoff - PLC Vendor
+Dianomic - IIoT Software
+Flir - IR/Gas Cameras
+General Atomics - Predator Drone
+Google - Search-ML-Cloud-TPUs
+JEA - Energy/Water Company
+[Motorsports.ai](http://motorsports.ai/) - Racing Digital Twins
+Nexcom - Industrial Gateways
+Nokia - Wireless Communications
```

Comment:
We can also add Nueman Aluminium could probably be added to the list as they have presented their use of Fledge at a TSC call

Reply:
Added Nueman Aluminium and signed commit.